go/pkg/bertyprotocol/store_message.go

```diff
@@ -147,10 +228,12 @@ func constructorFactoryGroupMessage(s *BertyOrbitDB) iface.StoreConstructor {
 		}
 
 		store := &messageStore{
-			devKS:  s.deviceKeystore,
-			mks:    s.messageKeystore,
-			g:      g,
-			logger: s.Logger(),
+			devKS:     s.deviceKeystore,
+			mks:       s.messageKeystore,
+			g:         g,
+			logger:    s.Logger(),
+			cache:     make(map[string][]ipfslog.Entry),
+			cacheChan: make(chan []byte, 10),
```

Comment:
nit: 10 is so small? how to determine this?

Reply:
This buffer is filled when:
1) a new chainkey is registered
2) a new message is put in the cache while the sender's chainkey is not registered yet

In a real context, these two events are rare, so I decided to put 10.

Reply:
is this limit OK for a conversation of >100 people?